src/control/ControlUtils.cpp

```diff
@@ -161,13 +34,13 @@ double ControlUtils::angleDifference(double A1, double A2) {
         angleDif -= 2 * M_PI;
     }
     return fabs(angleDif);
-}
+} // Code clone
```

Comment:
Should this comment be here or is this still a pending remove of code clone?

Reply:
This is indeed a code clone, but since it interferes with functions in ControlUtils that are not code clones I decided to remove it in the following up PR.